build-configs/tools/manage.ts

```diff
@@ -6,6 +6,8 @@ import fs from 'fs'
 import loadBuildConfig, { PlatformType } from '../index'
 
 const loadBuildConfigAsKeyValue = (buildConfigName: string, platform: PlatformType, spaces = true, quotes = false) => {
+  // If no buildConfig is found, that is handled in the the loadBuildConfig function
```

Comment:
This comment should be removed, as 
1) it is not really true, if no buildConfig is found an error is thrown, that is not really "handled"
2) if loadBuildConfig is changed somewhere in the future this comment will get outdated as no one can know an adjustment of this comment at this file is needed.